compose/common.yml

```diff
@@ -32,6 +32,13 @@ x-provider-server: &provider-server
     cardano-db-sync:
       condition: service_started
 
+x-provider-server-blockfrost: &provider-server-blockfrost
```

Comment:
As per the change in the `Dockerfile`, better to remove this.

Reply:
Docker-related changes are no longer a part of this PR.